File: backend/src/main/java/com/app/mevocab/internal/quiz/dto/request/QuestionDeleteRequest.java
package com.app.mevocab.internal.quiz.dto.request;

import com.app.mevocab.internal.quiz.dto.Question;
import lombok.*;
import lombok.experimental.FieldDefaults;

import java.util.List;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class QuestionDeleteRequest {
    String quizID;
    List<Integer> questionIDs;
}

File: frontend/lib/data/models/dto/Request/QuestionDeleteRequest.dart
class QuestionDeleteRequest {
  final String quizID;
  final List<int> questionIDs;

  QuestionDeleteRequest({
    required this.quizID,
    required this.questionIDs,
  });

  Map<String, dynamic> toJson() {
    return {
      'quizID': quizID,
      'questionIDs': questionIDs,
    };
  }
}
